src/main/java/br/com/alura/comex/config/security/SecurityConfiguration.java

```diff
@@ -0,0 +1,67 @@
+package br.com.alura.comex.config.security;
+
+import br.com.alura.comex.repository.UsuarioRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.ObjectPostProcessor;
+import org.springframework.security.config.annotation.authentication.configuration.AuthenticationConfiguration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+
+@Profile("dev")
+@Configuration
+public class SecurityConfiguration {
+
+    @Autowired
+    private AuthenticationService autenticacaoService;
+
+    @Autowired
+    private TokenService tokenService;
+
+    private UsuarioRepository usuarioRepository;
```

Comment:
Precisa injetar o repository de usuários senão dá NullPointerException!

```suggestion
    @Autowired
    private UsuarioRepository usuarioRepository;
```